### PR TITLE
Clarify that only labels are authoritative

### PR DESCRIPTION
**Pull request by marcoscaceres**
Add some clarifying text about labels being authoritative, rather than comments in issues. 

Make it clear that we may revise our position as new information becomes available.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ Please [file an issue](https://github.com/WebKit/standards-positions/issues/new)
 
 ### Possible positions
 
+**Please note:** Only once a label is assigned to an issue does it represent WebKit's position.
```
Comment (litherum) [inline README.md:11]: “Note” is a term-of-art in specs, and I think has the opposite meaning there than what you intend to mean here. I’d recommend removing the “please note” text.

Comment (marcoscaceres) [inline README.md:11]:
```suggestion
Only once a label is assigned to an issue does it represent WebKit's position.
```

Comment (marcoscaceres) [reply README.md]: Done.

**Review by litherum — APPROVED** [review]

**Review by hober — APPROVED** [review]
Works for me.

**Pull request merged**